install/common-pre-install/hpcm_installation-cpi.md

```diff
@@ -0,0 +1,1076 @@
+       ```
+
+   1. Inject these into the NCN images by running `ncn-image-modification.sh` from the CSM documentation RPM.
+
+       ```bash
+       NCN_MOD_SCRIPT=$(rpm -ql docs-csm | grep ncn-image-modification.sh)
+       echo "${NCN_MOD_SCRIPT}"
+       "${NCN_MOD_SCRIPT}" -p \
+          -d /root/.ssh \
+          -k "/var/www/ephemeral/data/k8s/${KUBERNETES_VERSION}/kubernetes-${KUBERNETES_VERSION}.squashfs" \
+          -s "/var/www/ephemeral/data/ceph/${CEPH_VERSION}/storage-ceph-${CEPH_VERSION}.squashfs"
+       ```
+
+1. (`pit#`) Log the currently installed PIT packages.
+
+   Having this information in the typescript can be helpful if problems are encountered during the install.
+   This command was run once in a previous step -- running it again now is intentional.
+
+   ```bash
+   /root/bin/metalid.sh
+   ```
+
+   Expected output looks similar to the following (the versions in the example below may differ). There should be __no__ errors.
+
+   ```text
+   = PIT Identification = COPY/CUT START =======================================
+   VERSION=1.6.0
+   TIMESTAMP=20220504161044
+   HASH=g10e2532
+   2022/05/04 17:08:19 Using config file: /var/www/ephemeral/prep/system_config.yaml
+   CRAY-Site-Init build signature...
+   Build Commit   : 0915d59f8292cfebe6b95dcba81b412a08e52ddf-main
+   Build Time     : 2022-05-02T20:21:46Z
+   Go Version     : go1.16.10
+   Git Version    : v1.9.13-29-g0915d59f
+   Platform       : linux/amd64
+   App. Version   : 1.17.1
+   metal-ipxe-2.2.6-1.noarch
+   metal-net-scripts-0.0.2-20210722171131_880ba18.noarch
+   metal-basecamp-1.1.12-1.x86_64
+   pit-init-1.2.20-1.noarch
+   pit-nexus-1.1.4-1.x86_64
+   = PIT Identification = COPY/CUT END =========================================
+   ```
+
+## Seed File generation
+
+The stepwise procedure to generate seed files is as follows:
+
+   1. Generate Paddle File
+      To generate the paddle file using the CANU Validate tool, follow link [Validate SHCD](../../operations/network/management_network/validate_shcd.md).
+
+      Example Command:
+
+      ```bash
+      canu validate shcd --shcd CrayInc-ShastaRiver-Groot-RevE12.xlsx --architecture V1 --tabs 40G_10G,NMN,HMN --corners I12,Q38,I13,Q21,J20,   U38 --json --out cabling.json
+      ```
+
+   1. Store SHCD Data in CVT Database.
+
+      ```bash
+      cm cvt parse shcd --canu_json_file cabling.json
+      ```
+
+      Example output:
+
+      ![store SHCD data](../../img/install/parse_shcd.PNG)
+
+   1. Create [`cabinets.yaml`](../create_cabinets_yaml.md) (Manually).
+
+   1. Capture hardware inventory and generate seed files and paddle file.
+
+      The following command stores the inventory of nodes and switches (fabric and management) in the database and generates the seed files and paddle file:
+
+      ```bash
+      cnodes | grep node >>nodelist
+      pdsh -w^nodelist /opt/clmgr/scripts/cluster-config-verification-tool/create_bond0.sh

```

Comment:
change the path to /opt/clmgr/**cluster-config-verification-tool/scripts**/create_bond0.sh